### ISSUE TITLE
Air Force One: clairvoyant air crew

### ISSUE THREAD

**Issue by ToxicFrog** [fgsfdsfgs/perfect_dark#341]
This I'm pretty sure is a vintage N64 bug.

When doing AF1, you need to go down into the cargo bay to get the evidence. This requires walking through the kitchen, with two crewmembers in it. Sometimes -- I'm not sure what makes the difference -- one of them will call you out as an imposter and from that point onwards your disguise is useless. It doesn't happen consistently, however, and sometimes you can just breeze right through the room with no issues.

**Comment by PeeMoldy** [comment]
Pro tip, those are bad guys who work for Trent Easton. You are allowed to eliminate the threat with your Laptop Gun. ;)

**Comment by RyanDwyer** [comment]
The same thing happens on the N64. I'm pretty sure whether they see you or not depends on whether they finish an animation with you in their line of sight.

They don't work for Trent. Rare just forgot to mark them as kill countable. Rare fixed it for the Japanese release, so killing them in the Japanese version will fail the mission.

**Comment by PeeMoldy** [comment]
> The same thing happens on the N64. I'm pretty sure whether they see you or not depends on whether they finish an animation with you in their line of sight.
> 
> They don't work for Trent. Rare just forgot to mark them as kill countable. Rare fixed it for the Japanese release, so killing them in the Japanese version will fail the mission.

Oh I thought they were conspirators because you can kill em in non Japanese version. That and one of them tries to sucker punch you.

**Comment by ToxicFrog** [comment]
Yeah, I always kind of figured they were not affiliated with Trent but also recognized me as not real air crew. The confusing part is that they don't _always_ notice, although if it's tied to animations completing that would explain it.
